heron/stmgr/src/cpp/manager/stmgr-client.cpp

```diff
@@ -199,7 +199,8 @@ void StMgrClient::SendStartBackPressureMessage() {
   REQID_Generator generator;
   REQID rand = generator.generate();
   // generator.generate(rand);
-  proto::stmgr::StartBackPressureMessage* message = acquire(message);
+  proto::stmgr::StartBackPressureMessage* message = acquire(
+    static_cast<proto::stmgr::StartBackPressureMessage *>(nullptr));
```

Comment:
I don't mind this. If we use your suggestion, somebody later might think it is needs to be cleaned up since a single assignment will suffice.